content/manuals/desktop/release-notes.md

```diff
@@ -23,6 +23,81 @@ Docker Desktop versions older than 6 months from the latest release are not avai
 
 Take a look at the [Docker Public Roadmap](https://github.com/orgs/docker/projects/51/views/1?filterQuery=) to see what's coming next.
 
+## 4.35.0
+
+{{< release-date date="2024-10-24" >}}
+
+{{< desktop-install-v2 all=true beta_win_arm=true version="4.35.0" build_path="/TBD/" >}}
+
+### New
+
+- Support for [Docker Desktop on Red Hat Enterprise Linux](/manuals/desktop/install/linux/rhel.md) is now generally available.
+- Volume Backup and Share is now generally available and can be found in the **Volumes** tab.
+- Terminal support within Docker Desktop using system shells is now generally available.
+- containerd image store:
+  - Improved output of `docker image inspect` to account for multi-platform images.
+  - Support multi-platform images via enabling users to pick a specific platform in `docker history`.
+- Beta release of Docker VMM - the faster alternative to Apple's Virtualization Framework on macOS (requires Apple Silicon and macOS 12.5 or later).
+
+### Upgrades
+
+- [containerd v1.7.21](https://github.com/containerd/containerd/releases/tag/v1.7.21)
+- [Docker Buildx v0.17.1](https://github.com/docker/buildx/releases/tag/v0.17.1)
+- [Docker Compose v2.29.7](https://github.com/docker/compose/releases/tag/v2.29.7)
+- [Docker Engine v27.3.1](https://docs.docker.com/engine/release-notes/27.3/#2731)
+- [Docker Scout CLI v1.14.0](https://github.com/docker/scout-cli/releases/tag/v1.14.0)
+- Docker Debug `v0.0.37`
+- Linux kernel `v6.10.9`
+
+### Bug fixes and enhancements
+
+#### For all platforms
+
+- Fixed a bug where proxy settings in `daemon.json` would override proxies set in Docker Desktop settings.
+- Fixed a bug where some Docker subnet ranges were not able to be used.
+- Added Docker VMM, a new powerful backend for creating and managing Docker Desktop Linux VM.
+- Removed [docker-index](https://github.com/docker/index-cli-plugin) as it is now deprecated, you can use `docker scout cves fs://<path to binary>` instead.
+- Fixed a bug where images couldn't be sorted or filtered by tag. Fixes [docker/for-win#14297](https://github.com/docker/for-win/issues/14297).
+- Fixed a bug where the `docker` CLI did not work as expected when the `registry.json` file was malformed.
+- Fixed a bug where the **Push to Docker Hub** action in the **Images** view would result in an `invalid tag format` error. Fixes [docker/for-win#14258](https://github.com/docker/for-win/issues/14258).
+- Fixed an issue where Docker Desktop startup failed when ICMPv6 setup was not successful.
+- Added drivers that allow USB/IP to work.
```

Comment:
@p1-0tr should this mention Linux, Mac, Hyper-V only? If it's a kernel change then WSL 2 users should check their kernel configuration?

Reply:
WSL2 should already support USB/IP out of the box. But, yes, we probably should call out Linux, Mac and Hyper-V, since it is not up to us at to what WSL2 does in this area.

Reply:
`Added drivers, on macOs, Linux and Hyper-V, that allow USB/IP to work.`